src/thunderbots/software/util/parameter/generate_parameters.py

```diff
@@ -220,7 +220,7 @@ def __header_and_cpp_gen(param_info: dict, cfg_name: str, header_file_pointer, c
                     name=key,
                     path=key,
                     namespace=cfg_name,
-                    default=parameter["default"],
+                    default=str(parameter["default"]).lower() if parameter["type"] == "bool" else parameter["default"],
```

Comment:
Quick comment why we're doing this would be good

Reply:
fixed